docs/source/tutorials/cascaded_architecture.rst

```diff
@@ -16,78 +16,251 @@ Download dataset
+                    },
+                    "FiLMedUnet": {
+                        "applied": false,
+                        "metadata": "contrasts",
+                        "film_layers": [0, 1, 0, 0, 0, 0, 0, 0, 0, 0]
+                    },
+                    "Modified3DUNet": {
+                        "applied": false,
+                        "length_3D": [128, 128, 16],
+                        "stride_3D": [128, 128, 16],
+                        "attention": false,
+                        "n_filters": 8
+                    },
+                    "uncertainty": {
+                        "epistemic": false,
+                        "aleatoric": false,
+                        "n_it": 0
+                    },
+                    "postprocessing": {
+                        "remove_noise": {"thr": -1},
+                        "keep_largest": {},
+                        "binarize_prediction": {"thr": 0.5},
+                        "uncertainty": {"thr": -1, "suffix": "_unc-vox.nii.gz"},
+                        "fill_holes": {},
+                        "remove_small": {"unit": "vox", "thr": 3}
+                    },
+                    "evaluation_parameters": {
+                        "target_size": {"unit": "vox", "thr": [20, 100]},
+                        "overlap": {"unit": "vox", "thr": 3}
+                    },
+                    "transformation": {
+                        "Resample":
+                        {
+                            "hspace": 0.75,
+                            "wspace": 0.75,
+                            "dspace": 1
+                        },
+                        "CenterCrop": {
+                            "size": [128, 128]},
+                        "RandomAffine": {
+                            "degrees": 5,
+                            "scale": [0.1, 0.1],
+                            "translate": [0.03, 0.03],
+                            "applied_to": ["im", "gt"],
+                            "dataset_type": ["training"]
+                        },
+                        "ElasticTransform": {
+                            "alpha_range": [28.0, 30.0],
+                            "sigma_range":  [3.5, 4.5],
+                            "p": 0.1,
+                            "applied_to": ["im", "gt"],
+                            "dataset_type": ["training"]
+                        },
+                      "NormalizeInstance": {"applied_to": ["im"]}
+                    }
+                }
+
+
+    From this point onward, we will discuss some of the key parameters to use cascaded models. Most parameters are configurable only via modification of the configuration ``JSON file``.
+    For those that supports commandline run time configuration, we included the respective command versions under the ``Command Line Interface`` tab
 
     - ``debugging``: Boolean, create extended verbosity and intermediate outputs. Here we will look at the intermediate predictions
```

Comment:
I would add a link to the `config.json` line number for each of these configuration options like here: 

![image](https://user-images.githubusercontent.com/6720068/150660508-db927fdf-f3e6-46dc-b706-57745c1569eb.png)

Make sure to update the URL to point to the correct line obviously.